ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Use this template for reporting bugs.
+---
+
+## Issue description
+
+If you found a bug or want to have a new feature implemented, please take your time and look whether this issue has been discussed in another issue or our mailing list already.
+
+SCM-Manager is open source. If you have any level of programming skills, maybe you can help yourself. We would love to support you.
```

Comment:
`any knowledge of programming` ?

Reply:
I dont think you need to add `yourself`. It sounds a bit like telling the user to not bother us and do it himself instead.